test-setup/src/Main.hs

```diff
@@ -14,8 +15,9 @@ main = do
     [] -> error "setup-tests expects one argument - the project directory whose code should be modified"
     xs -> modifyTests (head xs)
 
-hspecFormatterPath :: String
-hspecFormatterPath = "pre-compiled/test/HspecFormatter.hs"
+(|>) :: a -> b -> (a, b)
+(|>) = (,)
+infixl 4 |>
```

Comment:
I'm not a big Haskeller, but I've never seen this being used in Haskell code (I know the operator from F#, Elm and Elixir). @MatthijsBlom, your thoughts?

Reply:
[Hoogle: `|>`](https://hoogle.haskell.org/?hoogle=%7C%3E). As expected, I spot a few `flip (.)` synonyms and a lot of *cons*-likes, but no `(,)` synonyms. I have never previously seen this use of `|>` (and such) in Haskell.

This definition reminds me of Julia's [`=>`](https://docs.julialang.org/en/v1/base/collections/#Core.Pair).